### PR TITLE
Updated paragraph on Reporting Security Vulnerabilities

### PR DESCRIPTION
**Pull request by luanamulesoft**
on page: https://docs.mulesoft.com/runtime-manager/penetration-testing-policies

### CODE REVIEW

**Review by skediyal — DISMISSED** [review]
This looks good.

File: modules/ROOT/pages/penetration-testing-policies.adoc [hunk 1]
```diff
@@ -74,11 +74,21 @@ Include the following information in your email:
...
 ** All applicable HTTP requests and responses
 ** Explanation of why the example is considered a finding
-
-For more information about how to submit a security vulnerability finding, see the
-https://help.salesforce.com/articleView?id=000320207&type=1&mode=1[Security Vulnerability Finding Submittal Guide].
 --
 
+=== Reporting Security Vulnerabilities
+
+To report security vulnerability findings, please follow the https://help.salesforce.com/articleView?id=000320207&type=1&mode=1[Security Vulnerability Finding Submittal Guide]. Security vulnerability reports that do not appropriately follow the required steps will be rejected.
```
Comment (hannanelson) [inline modules/ROOT/pages/penetration-testing-policies.adoc]:
```suggestion
To report security vulnerability findings, follow the https://help.salesforce.com/articleView?id=000320207&type=1&mode=1[Security Vulnerability Finding Submittal Guide]. Security vulnerability reports that do not follow the required steps will be rejected.
```

File: modules/ROOT/pages/penetration-testing-policies.adoc [hunk 2]
```diff
@@ -74,11 +74,21 @@ Include the following information in your email:
...
+
+Please, note that Salesforce does not address any security vulnerabilities that fall under the following categories:
```
Comment (hannanelson) [inline modules/ROOT/pages/penetration-testing-policies.adoc]:
```suggestion
Salesforce does not address any security vulnerabilities in the following categories:
```

File: modules/ROOT/pages/penetration-testing-policies.adoc [hunk 3]
```diff
@@ -74,11 +74,21 @@ Include the following information in your email:
...
+
+* Scan output from automated vulnerability scanning tools without any valid proof of concept
+* Security bug without a valid proof of concept
```
Comment (hannanelson) [inline modules/ROOT/pages/penetration-testing-policies.adoc]:
```suggestion
* Security bugs without valid proof of concept
```

File: modules/ROOT/pages/penetration-testing-policies.adoc [hunk 4]
```diff
@@ -74,11 +74,21 @@ Include the following information in your email:
...
+* Vulnerabilities identified in an old or deprecated version of Mule runtime
```
Comment (hannanelson) [inline modules/ROOT/pages/penetration-testing-policies.adoc]:
```suggestion
* Vulnerabilities identified in old or deprecated versions of Mule runtime engine
```

File: modules/ROOT/pages/penetration-testing-policies.adoc [hunk 5]
```diff
@@ -74,11 +74,21 @@ Include the following information in your email:
...
+* Vulnerability induced via code customization made by your company
```
Comment (hannanelson) [inline modules/ROOT/pages/penetration-testing-policies.adoc]:
```suggestion
* Vulnerabilities introduced via code customization made by your company
```

**Review by hannanelson — DISMISSED** [review]

**Review by hannanelson — APPROVED** [review]

**Pull request merged**